src/main/java/uk/gov/justice/hmpps/prison/service/transformers/OffenderTransformer.java

```diff
@@ -80,8 +82,9 @@ public InmateDetail transform(final OffenderBooking latestBooking) {
                     .resultValue(pd.getCode() != null ? pd.getCode().getDescription() : pd.getProfileCode())
                     .build()).toList())
             .legalStatus(latestBooking.getLegalStatus())
-            .imprisonmentStatus(latestBooking.getActiveImprisonmentStatus().map(ims -> ims.getImprisonmentStatus() != null ? ims.getImprisonmentStatus().getStatus() : null).orElse(null))
-            .imprisonmentStatusDescription(latestBooking.getActiveImprisonmentStatus().map(ims -> ims.getImprisonmentStatus() != null ? ims.getImprisonmentStatus().getDescription() : null).orElse(null))
+            .imprisonmentStatus(imprisonmentStatus != null ? imprisonmentStatus.getStatus() : null)
+            .imprisonmentStatusDescription(imprisonmentStatus != null ? imprisonmentStatus.getDescription() : null)
+            .convictedStatus(imprisonmentStatus != null ? latestBooking.getConvictedStatus(): null)
```

Comment:
This changes InmateDetail Object (which is returned in other endpoints).  Still not sure whether I should add to Inmate detail or whether it should just be calculated separately.
All advise welcome